src/parser/Parser.ts

```diff
@@ -859,7 +795,7 @@ export class Parser {
             return this.ifStatement();
         }
 
-        if (this.check(TokenKind.Try)) {
+        if (this.check(TokenKind.Try) && this.checkAnyNext(TokenKind.Newline, TokenKind.Colon)) {
```

Comment:
It's to only take `try` as keyword when it's followed by `:` or newline immediately, otherwise it should be used as a variable. Now I see it should accept a comment as well.

Reply:
e.g. to support `try = true : print try`